From 48b40ba42158a23175a6b2b0448cd1c6b63d6953 Mon Sep 17 00:00:00 2001
From: xiao <x13945@163.com>
Date: Tue, 26 Jul 2016 02:04:32 +0800
Subject: [PATCH 67/73] android/ffmpeg:fix compile err while use ndk 11c.

I using the android ndk 11c as my ndk env. I got an error while execute
the "android/contrib/compile-ffmpeg.sh all", error message as:
You need the NDKr10e or later

Signed-off-by: Zhang Rui <bbcallen@gmail.com>
---
 android/contrib/tools/do-detect-env.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/android/contrib/tools/do-detect-env.sh b/android/contrib/tools/do-detect-env.sh
index 3101d26..93c3804 100644
--- a/android/contrib/tools/do-detect-env.sh
+++ b/android/contrib/tools/do-detect-env.sh
@@ -68,7 +68,7 @@ case "$IJK_NDK_REL" in
         IJK_NDK_REL=$(grep -o '^Pkg\.Revision.*=[0-9]*.*' $ANDROID_NDK/source.properties 2>/dev/null | sed 's/[[:space:]]*//g' | cut -d "=" -f 2)
         echo "IJK_NDK_REL=$IJK_NDK_REL"
         case "$IJK_NDK_REL" in
-            11|12*)
+            11*|12*)
                 if test -d ${ANDROID_NDK}/toolchains/arm-linux-androideabi-4.9
                 then
                     echo "NDKr$IJK_NDK_REL detected"
-- 
1.9.1

